### PR TITLE
Add class catalog filtering option placeholders

### PR DESCRIPTION
**Pull request by mvadari**
* Added search bar 
* Added "only open classes" toggle select

Test: 
* typed things into the search bar, hit "search", checked logs
* hit the "only open classes" toggle a bunch, checked logs

image

### CODE REVIEW

File: client/src/registration/ChangeClassesDashboard.tsx [hunk 1]
```diff
@@ -13,15 +14,14 @@ type Props = {
 type State = {
   catalog: Class[];
   enrolledClasses: string[];
+  openClasses: boolean;
```
Comment (evantey14) [inline client/src/registration/ChangeClassesDashboard.tsx:17]: I think I'd prefer if this were something about filtering or displaying  open classes? From this name, I would expect a Class array, not a boolean

File: client/src/registration/ChangeClassesDashboard.tsx [hunk 2]
```diff
@@ -13,15 +14,14 @@ type Props = {
...
+  search: string;
```
Comment (evantey14) [inline client/src/registration/ChangeClassesDashboard.tsx:18]: same deal earlier about a more descriptive name than just "search" I think even "searchQuery" would be better / make more sense (maybe this is just how I read it, but to me, search is more a verb than a noun?)

File: client/src/registration/ChangeClassesDashboard.tsx (hunk 1 above)
Comment (mvadari) [reply client/src/registration/ChangeClassesDashboard.tsx]: Done

File: client/src/registration/ChangeClassesDashboard.tsx (hunk 2 above)
Comment (mvadari) [reply client/src/registration/ChangeClassesDashboard.tsx]: done

**Pull request merged**